File: TFSWay.DomainModel/Entities/Reply.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.ComponentModel;
using System.ComponentModel.DataAnnotations;

namespace TFSWay.DomainModel.Entities
{
    public class Reply
    {
        [Key]
        public int ReplyId { get; set; }
        public int QueryId { get; set; }
        public string QueryText { get; set; }
        public DateTime? QueryDate { get; set; }
        public string ReplyText { get; set; }
        public DateTime? ReplyDate { get; set; }
        public DateTime? CreatedDate { get; set; }
        public string CreatedBy { get; set; }
    }
}

File: TFSWayUI/src/app/query/reply.ts
export interface IReply {

    replyId: number;
    QueryId: number;
    QueryText: string;
    QueryDate: Date;
    ReplyText: string;
    ReplyDate: Date;
    CreatedDate: Date;
    CreatedBy: string;


    replies: Array<IReply>;

}

